release_notes/ocp-4-15-release-notes.adoc

```diff
@@ -79,6 +79,13 @@ You can now install a cluster on {ibm-cloud-name} in an environment with limited
 
 // For more information see <insert links to restricted install assembly after docs merge>.
 
+[id="ocp-4-15-installation-and-update-nutanix-failure-domains"]
+==== Nutanix and fault tolerant deployments
+
+By default, the installation program installs control plane and compute machines into a single Nutanix Prism Element (cluster). To improve the fault tolerance of your {product-title} cluster, you can now specify that these machines be distributed across multiple Nutanix clusters by configuring failure domains.
```

Comment:
Just to be clear (and based on how I am interpreting this), "Prism Element" is Nutanix's version of a cluster? 
First time I am even aware of Nutanix, so not really sure.

Reply:
That is correct. In case you are interested. https://www.nutanixbible.com/3a-book-of-prism-architecture.html